arch/x86_64/mcount-dynamic.c

```diff
@@ -108,6 +108,8 @@ int mcount_setup_trampoline(struct mcount_dynamic_info *mdi)
 		       &dentry_addr, sizeof(dentry_addr));
 #endif
 	}
+
+	pr_dbg("Successfully set the trampoline at 0x%llx\n", mdi->trampoline);
```

Comment:
Please do not use `pr_dbg` for this kind message.  I think it's for debug level 2 or more - maybe not needed at all.